src/main/java/frc/robot/commands/arm/ArmFloorCMD.java

```diff
@@ -10,12 +11,19 @@ public class ArmFloorCMD extends SequentialCommandGroup {
 
   public ArmFloorCMD(Arm arm, IntakeRelease intakeRelease) {
     addCommands(
-        new ArmPositionCMD(
+        new ArmRotateCMD(
```

Comment:
If the arm is already extended when you run this, you'll be rotating it down without retracting it which could be unstable or hit the floor.  Could you do an ArmPositionCMD with extend=min(CONE_FLOOR.extendSetpoint, arm.currentPosition) instead?